tests/arch/arm/arm_irq_advanced_features/src/arm_irq_target_state.c

```diff
@@ -77,9 +77,19 @@ void test_arm_irq_target_state(void)
 		"Target state not set to Secure\n");
 	zassert_equal(irq_target_state_is_secure(i), 0,
 		"Target state not set to Non-Secure\n");
+
+	result_state =
+		irq_target_state_set(i, IRQ_TARGET_STATE_SECURE);
+
+	zassert_equal(result_state, IRQ_TARGET_STATE_SECURE,
+		"Target state not set to Secure\n");
+
+	zassert_equal(irq_target_state_is_secure(i), 1,
+		"Target state not set to Secure\n");
```

Comment:
what is going on with this change?

Reply:
The purpose is to make this test independent, otherwise run test_ arm_irq_target_state after test_arm_zero_latency_irqs  will cause problems.